.gitignore

```diff
@@ -44,4 +44,7 @@ public/workbox-*.js
 public/workbox-*.js.map
 
 # we use npm
-yarn.lock
\ No newline at end of file
+yarn.lock
+
+# delete this line if using Linux
+docker-compose.yml
```

Comment:
this is not necessary (I'll suggest something else in the `docker-compose.yml` file) and should be removed